### PR TITLE
Fixes #299:UI Enhancement in completed_scene.xml

### LINKED ISSUE #299: UI Enhancement in completed_scene.xml
**Suggestion**
Aligning the text content to the center and changing the background of **map** and **store** button(brown_button_background.png) will give the activity more cleaner look .
 
**Before**
whatsapp image 2017-02-01 at 1 15 19 am

**After**
device-2017-02-01-011447

@anubhakushwaha @aanchal07 @chhavip @yatna If this is an intended enhancement and if the UI looks good can I send the PR for this?Thanks :)

### PR DESCRIPTION
**Pull request by HimanshuS01**
Fixes #299 

**ScreenShots of change**

**In Portrait**
device-2017-02-01-131751
**In Landscape**
device-2017-02-01-131812

@yatna @anubhakushwaha @chhavip @aanchal07 please review this PR. Thanks :)

### CODE REVIEW

**Comment by aanchal07** [comment]
Good work @HimanshuS01 
@chhavip please see to this and merge this pr.

**Comment by HimanshuS01** [comment]
@aanchal07 Thank you so much for appreciating my work . :)

File: PowerUp/app/src/main/res/layout-land/completed_scene.xml [hunk 1]
```diff
@@ -18,23 +18,23 @@
...
+        android:layout_width="@dimen/ContinueButtonMap_width"
         android:layout_height="wrap_content"
         android:layout_alignParentBottom="true"
-        android:layout_alignParentEnd="true"
-        android:layout_alignParentRight="true"
-        android:layout_marginBottom="@dimen/completed_scene_continue_button_map_land_margin_bottom"
+        android:layout_alignLeft="@+id/completed_scene_textView"
+        android:background="@drawable/brown_button_background"
+        android:layout_marginBottom="@dimen/ContinueButtonMap_margin_bottom_landscape"
         android:layout_marginRight="@dimen/completed_scene_continue_button_map_land_margin_right"
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/res/layout-land/completed_scene.xml]: start with small letters in all the id's and tag names for resources taken from dimen

File: PowerUp/app/src/main/res/layout-land/completed_scene.xml [hunk 2]
```diff
@@ -18,23 +18,23 @@
...
         android:text="@string/map" />
 
     <Button
         android:id="@+id/store_button"
-        android:layout_width="wrap_content"
+        android:layout_width="@dimen/StoreButtonMap_width"
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/res/layout-land/completed_scene.xml]: Here as well

File: PowerUp/app/src/main/res/layout/completed_scene.xml [hunk 3]
```diff
@@ -19,21 +18,25 @@
 
     <Button
         android:id="@+id/ContinueButtonMap"
-        android:layout_width="wrap_content"
+        android:layout_width="@dimen/ContinueButtonMap_width"
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/res/layout/completed_scene.xml]: Here also

File: PowerUp/app/src/main/res/layout/completed_scene.xml [hunk 4]
```diff
@@ -19,21 +18,25 @@
...
+        android:layout_alignLeft="@+id/completed_scene_textView"
+        android:background="@drawable/brown_button_background"
+        android:layout_marginBottom="@dimen/ContinueButtonMap_margin_bottom"
+        android:layout_marginRight="@dimen/completed_scene_continue_button_map_land_margin_right"
         android:text="@string/map" />
 
     <Button
         android:id="@+id/store_button"
-        android:layout_width="wrap_content"
+        android:layout_width="@dimen/StoreButtonMap_width"
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/res/layout/completed_scene.xml]: Here again

File: PowerUp/app/src/main/res/values/dimens.xml [hunk 5]
```diff
@@ -166,6 +166,10 @@
     <dimen name="button_layout_width">160dp</dimen>
     <dimen name="replay_layout_marginTop">365dp</dimen>
     <dimen name="margin_left_replay_button">30dp</dimen>
-
+    <dimen name="ContinueButtonMap_width">100dp</dimen>
```
Comment (anubhakushwaha) [inline PowerUp/app/src/main/res/values/dimens.xml]: All the possible places should be changed adhering to coding guidelines.

**Comment by anubhakushwaha** [comment]
@HimanshuS01 Further as per the consistency for the rest of the app try making the textSize same as the rest of buttons i.e 18sp else the work is good.

**Comment by HimanshuS01** [comment]
@anubhakushwaha  Sure.I got your point and will take care of it from the next time. :)

**Comment by HimanshuS01** [comment]
@chhavip @aanchal07 @anubhakushwaha changes done. Please review :)

**Comment by aanchal07** [comment]
@chhavip Looks good to me. Can be merged.

**Comment by chhavip** [comment]
Merging. Thanks @HimanshuS01

**Pull request merged**